static/js/components/admin/EditChannelAppearanceForm.js

```diff
@@ -50,7 +50,7 @@ export default class EditChannelAppearanceForm extends React.Component<Props> {
                 form.avatar.edit &&
                 URL.createObjectURL(form.avatar.edit)
               }
-              imageSize="medium"
+              imageSize="large"
```

Comment:
Maybe change to:
```python
imageSize=CHANNEL_AVATAR_LARGE
```

`CHANNEL_AVATAR_LARGE` was added in `ChannelAvatar.js`, doesn't seem to be referenced anywhere else but this might be a good place to make use of it.

Reply:
Actually never mind, glanced over that part of the code too quickly, it's fine.